### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (86)

### CODE REVIEW

File: stylesheets/components/toast.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+     font-size: larger;
+   }
+
+.box {
+     display: flex;
+     flex-direction: row;
+     justify-content: center;
+     align-items: center;
+     text-align: center;
+    justify-content: center;
```
Comment (sadathshariff) [inline stylesheets/components/toast.css]: you are using flex and text-align: center and by default flex-direction is row any specific reason here for using it.

**Review by kjais1720 — APPROVED** [review]
Looks good overall👍

File: stylesheets/components/toast.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import url(/stylesheets/main.css);
+
+.toast-msg {
+     padding: 0.5rem 1rem;
+     background-color: var(--white);
+     border: 0.1rem solid var(--black);
+     font-size: larger;
```
Comment (sadathshariff) [inline stylesheets/components/toast.css]: Try to have a variable or give value instead of directly specifying larger

**Review by sadathshariff — COMMENTED** [review]
Suggested some small changes rest everything looks good, make sure to check the contrast of the colors

File: stylesheets/components/toast.css (hunk 2 above)
Comment (iamsaurav28) [reply stylesheets/components/toast.css]: ok done

File: stylesheets/components/toast.css (hunk 1 above)
Comment (iamsaurav28) [reply stylesheets/components/toast.css]: oh ya ty. i ll delete it.

**Review by sadathshariff — APPROVED** [review]

**Pull request merged**